docs/source/science/ATL08/ATL08.ipynb

```diff
@@ -0,0 +1,1389 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    with s3_fsspec.open(nsidc_object) as f:
</pre><p>Do you need this context manager? It seems like it might be possible to just use the s3_url as the path directly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/362/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>So far it doesn't seem to want to work without it, but it could also easily be me just not setting it up right - I'll look into it some more!</p>

Reply:
<p>Ohh wait, unless you meant nsidc_object and s3_url. I went ahead and updated to directly use the s3_url! Ignore my above comment</p>